### PR TITLE
Avoid ServeApi race condition

### PR DESCRIPTION
**Pull request by LK4D4**
Closes #11944

### CODE REVIEW

**Comment by jfrazelle** [comment]
is this needed in 1.6.0

**Comment by LK4D4** [comment]
@jfrazelle @ibuildthecloud want it hard.

File: api/server/server.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ import (
 )
 
 var (
-	activationLock chan struct{}
+	activationLock chan struct{} = make(chan struct{})
```
Comment (tiborvass) [inline api/server/server.go]: @LK4D4 type can be inferred :)

Comment (LK4D4) [inline api/server/server.go]: Don't wanna touch @ibuildthecloud commit :) Let's create issue for contributors about this later.

Comment (tiborvass) [inline api/server/server.go]: haha, your joker sentence

**Comment by jfrazelle** [comment]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**